kura/emulator/org.eclipse.kura.emulator/src/main/resources/kura.properties

```diff
@@ -25,6 +26,7 @@ kura.partNumber=DevPartNumber
 kura.serialNumber=DevSerialNumber
 kura.bios.version=DevBiosVersion
 kura.firmware.version=DevFirmwareVersion
+kura.primary.network.interface=eth0
```

Comment:
I believe this property is ignore on the Mac which always use en0 :-( Please add a comment.

Reply:
Done